code/game/objects/items/devices/drilldroptargetter.dm

```diff
@@ -0,0 +1,117 @@
+/obj/item/device/drill_dropper
+	name = "laser targeting dropper"
+	desc = "Your drill shall pierce the heavens."
```

Comment:
this reference is so overused on drills that thermal drills already make fun of this reference

Reply:
honestly even just going with "This drill literally pierces the heavens." would be an improvement

Reply:
The sad cowboy is right.